src/components/SpecialResolution/SpecialResolutionForm.vue

```diff
@@ -0,0 +1,463 @@
+import { Component, Mixins, Watch } from 'vue-property-decorator'
+import { Action, Getter } from 'vuex-class'
+import { ActionBindingIF, HelpSectionIF, ResourceIF,
+  FormIF, PersonIF, EmptyPerson, CreateResolutionIF } from '@/interfaces/'
+import { DateMixin, SharedMixin, FilingTemplateMixin, CommonMixin } from '@/mixins/'
+import { HelpSection } from '@/components/common/'
+import { DatePicker as DatePickerShared } from '@bcrs-shared-components/date-picker/'
+
+@Component({
+  components: {
+    HelpSection,
+    DatePickerShared
+  }
+})
+export default class SpecialResolutionForm extends Mixins(
+  DateMixin
+
+) {
+  @Getter getResource!: ResourceIF
+  @Getter getBusinessFoundingDate!: Date
+
+  @Action setResolution!: ActionBindingIF
+  @Getter getCreateResolutionStep!: CreateResolutionIF
+  @Getter getComponentValidate!: boolean
+
+  // Refs
+  $refs!: {
+    resolutionDatePickerRef: DatePickerShared,
+    signatureDatePickerRef: DatePickerShared,
+    signingPersonFormRef: FormIF,
+  }
+
+  // Date properties
+  protected resolutionDateText: string = ''
+  protected signatureDateText: string = ''
+
+  protected MAX_RESOLUTION_TEXT_LENGTH: number = 1000
+  protected resolutionText: string = ''
+
+  protected signingPerson: PersonIF = null
+
+  get helpSection (): HelpSectionIF {
+    return this.getResource.changeData?.specialSpecialResolution?.helpSection
+  }
+
+  get getCreateResolutionResource (): any {
+    return this.getResource.changeData?.specialSpecialResolution?.sampleFormSection || null
+  }
+
+  get documentURL (): string {
+    const docUrl = sessionStorage.getItem('BASE_URL') +
+      this.getCreateResolutionResource?.downloadDocPath
+    return docUrl
+  }
+
+  get previewImageSource (): string {
+    // Note: the image file path did not resolve correctly when using the require function directly.  In order
+    // to get the image path resolving correctly, needed to get the image context first and use that to build
+    // the final image file path
+
+    if (this.isJestRunning || !this.getCreateResolutionResource?.previewImagePath) { return '' }
+    const images = require.context('@/assets/images/', false, /\.png$/)
+    return images('./' + this.getCreateResolutionResource?.previewImagePath)
+  }
+
+  /** The name section validity state (when prompted by app). */
+  get invalidNameSection (): boolean {
+    // add more state here
+    return (this.getComponentValidate)
+  }
+
+  /** The minimum date that can be entered (can't be earlier than incorporation date ). */
+  get resolutionDateMinStr (): string {
+    return this.dateToYyyyMmDd(new Date(this.getBusinessFoundingDate))
+  }
+
+  /** The minimum date that can be entered (can't be earlier than incorporation date ). */
+  get resolutionDateMin (): Date {
+    return this.getBusinessFoundingDate
+  }
+  /** The maximum date that can be entered (today). */
+  get resolutionDateMax (): Date {
+    const date = new Date()
```

Comment:
Again, don't do this. You don't know what date you'll get.

There's a getter for current date (yyyy-mm-dd) already. (And it is updated periodically in the background.)